src/components/GenericErrorView/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {View} from 'react-native';
+import Icon from '../Icon';
+import defaultTheme from '../../styles/themes/default';
+import * as Expensicons from '../Icon/Expensicons';
+import ExpensifyText from '../ExpensifyText';
+import Button from '../Button';
+import LogoWordmark from '../../../assets/images/expensify-wordmark.svg';
+import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../withWindowDimensions';
+import compose from '../../libs/compose';
+import variables from '../../styles/variables';
+import Navigation from '../../libs/Navigation/Navigation';
+import styles from '../../styles/styles';
+import ErrorBodyText from './ErrorBodyText';
+import TextLink from '../TextLink';
+import CONST from '../../CONST';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+    ...windowDimensionsPropTypes,
+};
+
+const GenericErrorView = (props) => {
+    const refreshPage = () => {
+        Navigation.navigate(Navigation.getCurrentRoute());
+    };
+
+    return (
+        <View style={[styles.flex1, styles.pv10, styles.ph5, styles.errorPageContainer]}>
+            <View style={[styles.flexColumn, styles.flex1, styles.alignItemsCenter, styles.justifyContentCenter]}>
+                <View style={[styles.flexColumn, styles.flexColumn, styles.alignItemsStart]}>
+                    <View style={[styles.mb5]}>
```

Comment:
```suggestion
                    <View style={styles.mb5}>
```

Reply:
FWIW, I stopped making this suggestion. I used to try and be consistent about this if there was only one style, but AFAIK there's no harm in having styles be an array, is there? In fact, we could even start being consistent about styles _always_ being an array. Kinda reminds me of how we enforce trailing commas – to make it easier to add new stuff in.